homeassistant/components/automation/__init__.py

```diff
@@ -101,6 +101,7 @@
 ATTR_SOURCE = "source"
 ATTR_VARIABLES = "variables"
 SERVICE_TRIGGER = "trigger"
+ATTR_INPUTS = "inputs"
```

Comment:
Move this before `ATTR_LAST_TRIGGERED` to better group things and keep them in alphabetical order.

Reply:
Done :+1: